### ISSUE TITLE
Use git-lfs in data repository

### ISSUE THREAD

**Issue by zperova** [PDXFinder/pdxfinder#177]
## # Description

The data repository is exceeding the size limit for the internal EBI gitlab (5GB)

We need to use to git-lfs manage the files and store the objects in a server that will not have size restrictions.

Your query has been assigned an id of [www-prod #430772](https://helpdesk.ebi.ac.uk/NoAuth/Login.html?next=0932df17bc37608f08efe39eb61991bc). Please include this string in any related emails.

You can check the status of your ticket by clicking on https://helpdesk.ebi.ac.uk/Ticket/Display.html?id=430772

### Acceptance Criteria
- [ ] Store all of the *.tsv, *xlsx, and *.json files in a git-lfs server, preferably on the EBI gitlab server
- [ ] Contact www-prod to enable this

**Comment by zperova** [comment]
Size of data repository has been increased to 20 GB - both the pdxfinder data repo and pdxfinder-lfs
However, there are problems with the lfs repo.

Currently what is needed is to investigate what exactly is the problem

**Comment by Afollet** [comment]
There appears to be an error when git clone is used. 

```
Error downloading object: data/JAX/cna/J000077451.json (f1e7d1e): Smudge error: Error downloading data/JAX/cna/J000077451.json (f1e7d1e9ad2542f805a24d81930d209c026ee000a5a34b2dd52ed97f53d2bc00): open /home/afollette/Finder_Data_Repositories/pdxfinder-lfs/.git/lfs/objects/incomplete/f1e7d1e9ad2542f805a24d81930d209c026ee000a5a34b2dd52ed97f53d2bc00.tmp: too many open files
```
See attached document for full error output [https://app.zenhub.com/files/78657127/5b625f3f-96a7-4e21-a873-8671c964a2e1/download](https://app.zenhub.com/files/78657127/5b625f3f-96a7-4e21-a873-8671c964a2e1/download)

**Comment by Afollet** [comment]
Further more, when running 

```
git lfs pull
```
Lfs complains about the downgrading of https to http and a post to a non-existant host 172.21.1.100:53[https://app.zenhub.com/files/78657127/34fdc9af-21a1-4630-b484-3e50b63ff0e0/download](https://app.zenhub.com/files/78657127/34fdc9af-21a1-4630-b484-3e50b63ff0e0/download)

**Comment by rmwthorne** [comment]
Blocked by storage procurement, detailed in #205. Once that is resolved this issue can be resolved. In the meantime git-lfs has ben disabled for pdxfinder-data.

**Comment by Afollet** [comment]
Can you confirm that you disable git-lfs for pdxfinder-data/release-2020-04? This branch still has lsf pointers for .xlsx files and is currently still the pdxFinders development branch.

**Comment by rmwthorne** [comment]
Sorry, didn't realise you were still using `release-2020-04`. Undoing the the git-lfs commits were difficult as I couldn't check out the branch, instead, the relevant commits from release-2020-04 (basically all those except those enabling git-lfs) were merged into `master`. If you delete this branch and set PDXFinder's development branch to `master` (or `dev`), all should be sorted.

**Comment by zperova** [comment]
@zperova need to check with systems on when they will implement this.
